test/integration/canonicalize/canonical.expected

```diff
@@ -50,42 +50,31 @@ model {
 generated quantities {
   real y0[2] = {1.0, 2.0};
   real ts[3] = {0.5, 1.0, 2.0};
-  real y_hat[3, 2] = integrate_ode_rk45(sho, y0, 0.0, ts, theta, x_r, x_i);
+  real y_hat[3, 2] = ode_rk45(sho, y0, 0.0, ts, theta, x_r, x_i);
 }
 
 
-Warning: deprecated language construct used in 'deprecated.stan', line 55, column 20 to column 69:
-integrate_ode is deprecated, use integrate_ode_rk45 instead
+Warning: in 'deprecated.stan', line 55, column 20 to column 69: integrate_ode is deprecated and will be removed in the future. Use ode_rk45 instead
 
-Warning: deprecated language construct used in 'deprecated.stan', line 28, column 11 to column 46:
-The function `if_else` is deprecated. Use the conditional operator (x ? y : z) instead.
+Warning: in 'deprecated.stan', line 28, column 11 to column 46: The function `if_else` is deprecated. Use the conditional operator (x ? y : z) instead.
 
-Warning: deprecated language construct used in 'deprecated.stan', line 28, column 24 to column 42:
-multiply_log is deprecated, use lmultiply instead
+Warning: in 'deprecated.stan', line 28, column 24 to column 42: multiply_log is deprecated and will be removed in the future. Use lmultiply instead
```

Comment:
Minor: Some of these warnings end with a period and some don't.

Reply:
Oops! Any preference? What would be more consistent with other places?

Reply:
Huh, I do not have a preference. I am also not huge friend with punctuation in such cases haha. My gut feeling is period everywhere but not super confident.

Reply:
Periods galore.